src/ILCompiler.ReadyToRun/src/Compiler/ReadyToRunMetadataFieldLayoutAlgorithm.cs

```diff
@@ -741,12 +743,12 @@ public static bool IsManagedSequentialType(TypeDesc type)
             {
                 foreach (FieldDesc field in type.GetFields())
                 {
-                    if (!field.IsStatic && !field.IsLiteral)
+                    if (!field.IsStatic &&
+                        !field.IsLiteral &&
```

Comment:
`!field.IsStatic && !field.IsLiteral` can be reduced to `!field.IsStatic`. Literal instance fields don't exist.

Reply:
Fixed in 2nd commit.